### ISSUE TITLE
Add timeout to automatically select history again instead of selected collection

### ISSUE THREAD

**Issue by SimonMayerhofer** [IndieGoodies/PastePal#78]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I want to paste a clipboard entry but notice that I selected a collection an hour ago, which is still active.
I then need to first click on history to be able to paste one of the entries.

**Describe the solution you'd like**
Defining a timeout in the settings when PastePal automatically goes back to history and not having a collection open (in the side window)

**Comment by onmyway133** [comment]
@SimonMayerhofer hi thanks for reaching out. To understand the issue better, do you want to select and copy/paste the clip in the history or in that collection?

**Comment by SimonMayerhofer** [comment]
I want to select the clip in the history. But when the collection was selected before it stays active until you manually select history again.

**Comment by onmyway133** [comment]
@SimonMayerhofer Hi, when we select a collection, it is active, this is helpful when people want to work on saved clips in that collection alone. So when they open the side window, that collection is active. 

If I understand correctly, you want the side window to select history again when it is opened?

**Comment by SimonMayerhofer** [comment]
> If I understand correctly, you want the side window to select history again when it is opened?

Yes, after a configurable delay. So e.g. after 5/10/15/etc. minutes select history again.

**Comment by buildwithdon** [comment]
Yes, this would be very helpful...

During normal workflow... not all clips are in a single collection. I would like it to go to the recent history by default after I've already used the clip from the collection. A toggle in the settings would be amazing.
